### PR TITLE
Install workflows - textual updates based on doc review

### PR DESCRIPTION
**Pull request by SanjalKatiyar**
### Before:
Screenshot from 2021-02-25 14-54-43
Screenshot from 2021-02-25 14-55-05
Screenshot from 2021-02-25 14-56-53
Screenshot from 2021-02-25 14-57-04

### After:
Screenshot from 2021-02-25 15-07-58
Screenshot from 2021-02-25 15-16-33
Screenshot from 2021-02-25 15-17-27
Screenshot from 2021-02-25 15-17-37
Screenshot from 2021-02-25 15-18-19

### Textual update for taint nodes checkbox is handled in https://github.com/openshift/console/pull/8218
Screenshot from 2021-02-25 15-03-44

--------------------------------------------------------------------------------------------------------------------------------------------------------------------------

### After Edit 1: 
Screenshot from 2021-03-09 10-49-54
Screenshot from 2021-03-09 10-51-40

--------------------------------------------------------------------------------------------------------------------------------------------------------------------------

### Edit 2: Text under "Selected Nodes" for step-3 of LSO mode now looks like this: https://github.com/openshift/console/pull/9019 

--------------------------------------------------------------------------------------------------------------------------------------------------------------------------

### CODE REVIEW

**Comment by cloudbehl** [comment]
please add a screenshot of before and after so that the UX team can review it.

**Comment by SanjalKatiyar** [comment]
> please add a screenshot of before and after so that the UX team can review it.

**Comment by SanjalKatiyar** [comment]
> please add a screenshot of before and after so that the UX team can review it.

okay... got it...

File: frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install-wizard.tsx [hunk 1]
```diff
@@ -222,7 +222,7 @@ const CreateStorageClusterWizard: React.FC<CreateStorageClusterWizardProps> = ({
       name: (
         <Link to={toLink(CreateStepsSC, CreateStepsSC.STORAGECLASS, MODES)}>
           {' '}
-          {t('ceph-storage-plugin~Create Storage Class')}{' '}
+          {t('ceph-storage-plugin~Create storage class')}{' '}
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/install-wizard.tsx:225]:
```suggestion
          {t('ceph-storage-plugin~Create StorageClass')}{' '}
```

File: frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/body.tsx [hunk 2]
```diff
@@ -85,17 +87,15 @@ export const LocalVolumeSetBody: React.FC<LocalVolumeSetBodyProps> = ({
         />
       </FormGroup>
       <Text component={TextVariants.h3} className="lso-create-lvs__filter-volumes-text--margin">
-        {t('lso-plugin~Filter Disks')}
+        {t('lso-plugin~Filter Disks By')}
```
Comment (bipuladh) [inline frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/body.tsx:90]:
```suggestion
        {t('lso-plugin~Filter Disks by')}
```

**Review by bipuladh — CHANGES_REQUESTED** [review]

**Comment by yuvalgalanti** [comment]
It looks good, I wouldn't change the way we writing "Storage Class" for now. lets do a revisit for all textual alignments in a new PR

**Comment by SanjalKatiyar** [comment]
> It looks good, I wouldn't change the way we writing "Storage Class" for now. lets do a revisit for all textual alignments in a new PR

@yuvalgalanti @bipuladh ... FYI, several string related updates (for example: "storage class" --> "StorageClass") were handled in this: https://github.com/openshift/console/pull/9017 (already merged)...

**Pull request merged**